src/firewall/functions.py

```diff
@@ -95,6 +95,12 @@ def addr_family_str(family):
 ###############################################################################
 
 
+IPAddrZero4 = b"\0\0\0\0"
+
+
+###############################################################################
+
+
```

Comment:
functions.py is more than 1k lines of code. Which is perfectly fine, because it's just a heap of independent utilities and you can understand parts independently (you don't need to have it all at once in your head). However, it's still useful to have visual clues about things that belong together (which which don't).

The value is the visual clue about related/unrelated code and the benefits when reading the code.

What are the problems with them?